sdk/communication/azure-communication-callingserver/src/main/java/com/azure/communication/callingserver/models/events/CallingServerBaseEvent.java

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.communication.callingserver.models.events;
+
+/** The base event interface. */
+public interface CallingServerBaseEvent {
+    /**
+     * Get the type property: The event type.
+     *
+     * @return the type value.
+     */
+    AcsEventType getType();
```

Comment:
ServerCallId, CallConnectionId, and CorrelationId are common among events. Should those be part of base event here?

Reply:
Yes, I will move them.

Reply:
fixed